submissions/mustbefail/dom-api-task/src/js/app.js

```diff
@@ -5,50 +5,55 @@ const makeMsg = (ticketsAmount, totalPrice) => {
   return `You have selected ${ticketsAmount} seats for a price of $${totalPrice}`;
 };
 
-const render = ({ ticketsAmount, totalPrice }) => {
+const renderApp = ({ ticketsAmount, totalPrice }) => {
   const cartTip = document.querySelector('.add-to-cart-block p');
   cartTip.textContent = makeMsg(ticketsAmount, totalPrice);
 };
 
-export default () => {
+const app = () => {
+  const form = document.querySelector('form');
+  const modal = document.querySelector('.modal');
+  const addToCartBtn = document.querySelector('.add-to-cart-block button');
+
+  const toggleModal = () => {
+    modal.classList.toggle('show-modal');
+  };
+
   const state = {
     ticketPrice: 24,
     cart: [],
     totalPrice: null,
     ticketsAmount: null,
   };
 
-  const form = document.querySelector('form');
   form.addEventListener('click', (event) => {
     const { currentTarget } = event;
 
     const seats = currentTarget.querySelectorAll('input:checked');
-    const seatsData = Array.from(seats).map((seat) => seat.value);
+    const seatsData = [...seats].map((seat) => seat.value);
 
     state.cart = seatsData;
     state.ticketsAmount = seatsData.length;
     state.totalPrice = state.ticketsAmount * state.ticketPrice;
-    render(state);
+    renderApp(state);
   });
 
-  const modal = document.querySelector('.modal');
-
-  const toggleModal = () => {
-    modal.classList.toggle('show-modal');
-  };
-
-  const addToCartBtn = document.querySelector('.add-to-cart-block button');
   addToCartBtn.addEventListener('click', (event) => {
     event.preventDefault();
 
     modal.innerHTML = makeModal(state);
     toggleModal();
   });
 
+  document.addEventListener('keydown', ({ keyCode }) => {
+    if (keyCode === 27) toggleModal();
+  });
+
   modal.addEventListener('click', (event) => {
     const closeBtn = modal.querySelector('.close-button');
-
     if (event.target === modal) toggleModal();
     if (event.target === closeBtn) toggleModal();
   });
 };
+
+export default app;
```

Comment:
[Read more](https://stackoverflow.com/questions/33611812/export-const-vs-export-default-in-es6) about export default vs named export.

Reply:
that is, it is better not to use the default export because we do not know what we are importing and this can lead to errors, am I right?

Reply:
Yes, Also, you can accidentally rename an imported entity and have a problem with it.)